readme.md

```diff
@@ -170,6 +170,24 @@ import { config } from "favicons";
 
 For the full list of files, check `config/files.json`. For the full HTML code, check `config/html.js`. Finally, for the full list of icons, check `config/icons.json`.
 
+To save the files that are output, you can use `fs`.
+
+For example, amend the code above for Node.js as follows:
+
+```
+var fs = require ("fs");
```

Comment:
`const` is preferable over `var`.

Reply:
While we support CommonJS modules, it's better to promote ECMAScript modules.

Reply:
I updated the PR to use ECMAScript.